.github/workflows/docker.yml

```diff
@@ -53,5 +36,39 @@ jobs:
         run: |
           docker run --privileged --rm tonistiigi/binfmt --install arm64,amd64
           docker buildx create --use --name cross-builder
-      - name: Build and push ${{ matrix.build.name }}
-        run: ${{ matrix.build.command }}
+
+  reth:
+    needs: setup
+    runs-on: ubuntu-24.04
+    if: ${{ contains(github.ref, '-rc') }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build and push reth
+        run: make IMAGE_NAME=$IMAGE_NAME DOCKER_IMAGE_NAME=$DOCKER_IMAGE_NAME PROFILE=maxperf docker-build-push
```

Comment:
state does not carry over between jobs, so the docker setup + login + cache setup does not carry over from `setup` to e.g. `reth`